pages/deployment/aws.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Deploy Memgraph on AWS EC2 instance 
+description: Learn how to deploy your Memgraph instance on the AWS EC2 instance.
+---
+
+import { Card, Cards, Tabs, Tab } from 'nextra/components'
+
+# Deploy Memgraph in AWS EC2 instance
+
+This guide will show you how to deploy Memgraph on an AWS EC2 instance. The
+guide will cover only the specific bits that are different from the general
+deployment guide you can find for native [linux](./linux.mdx) or
+[docker](./docker.mdx) deployments.
+
+This guide assumes you have an AWS account and are familiar with the AWS
+console. If you are not, you can follow the [AWS EC2
+documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html).
+
+
+## Crating a new EC2 instance 
+
+The first step is to have an EC2 instance running on AWS, where you will deploy
+Memgraph. Here are some guidelines for opening a new EC2 instance for memgraph. 
+
+### Picking the OS 
+
+When creating the EC2 instance, you need to pick the OS you wish to have on the
+instance. If you are going to use the Memgraph native installation, you need to
+pick a Linux distribution that Memgraph supports.
+
+Memgraph supports the following Linux distributions:
+
+- Ubuntu (20.04, 22.04)
+- Fedora (38, 39)
+- Debian (11, 12) 
+- Centos (7, 9) 
+- Amazon Linux (2)
+- RedHat (7, 9)
+- Rocky (9.3)
```

Comment:
I would be nice to have this listed somewhere else (e.g. Installation), so it can be updated only on that page in the future (and not on every one where it's mentioned). @kgolubic Do we have such page or section where all supported Linux distributions are listed?

Reply:
@katarinasupe we have a page with direct download links -> https://memgraph.com/docs/getting-started/install-memgraph/direct-download-links. If needed I can create a such page, or simlply add a paragraph to the top of direct download page that these are the supported Linux distributions.

Reply:
I have updated this